libs/community/langchain_community/callbacks/mlflow_callback.py

```diff
@@ -140,89 +170,83 @@ def __init__(self, **kwargs: Any):
             )
             self.mlflow.set_tracking_uri(tracking_uri)
 
-            # User can set other env variables described here
-            # > https://www.mlflow.org/docs/latest/tracking.html#logging-to-a-tracking-server
-
-            experiment_name = get_from_dict_or_env(
-                kwargs, "experiment_name", "MLFLOW_EXPERIMENT_NAME"
-            )
-            self.mlf_exp = self.mlflow.get_experiment_by_name(experiment_name)
-            if self.mlf_exp is not None:
-                self.mlf_expid = self.mlf_exp.experiment_id
+            if kwargs.get("run_id", None) is not None:
```

Comment:
```suggestion
            if "run_id" in kwargs:
```

Reply:
```suggestion
            if run_id := kwargs.get("run_id"):
```

or assignment expression